frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -52,7 +58,61 @@ import {
   getPort,
 } from '../kms-config/utils';
 import './ocs-storage-class-form.scss';
-import { GUARDED_FEATURES } from '../../features';
+
+export const CephFsNameComponent: React.FC<ProvisionerProps> = ({ onParamChange }) => {
+  const { t } = useTranslation();
+
+  const isExternal = useFlag(OCS_INDEPENDENT_FLAG);
+  const scName = `${isExternal ? OCS_EXTERNAL_CR_NAME : OCS_INTERNAL_CR_NAME}-cephfs`;
+  const [sc, scLoaded, scLoadError] = useK8sGet<StorageClassResourceKind>(
+    StorageClassModel,
+    scName,
+  );
+  const [fileSystemName, setFileSystemName] = React.useState<string>('');
```

Comment:
```suggestion
```
I don't think we need to maintain state at two different places.
We should make changes in the API at https://github.com/openshift/console/blob/master/frontend/public/components/storage-class-form.tsx#L863-L866
```
<Component
            key={key}
            onParamChange={(value: string) => this.setParameterHandler(key, value, false)}
            value={this.state.newStorageClass.params[key]}
          />
```